stabilizer/src/test/java/com/hazelcast/stabilizer/AgentSmokeTest.java

```diff
@@ -150,7 +155,7 @@ public void run() {
     private AgentsClient getClient() throws IOException {
         File agentFile = File.createTempFile("agents", "txt");
         agentFile.deleteOnExit();
-        Utils.writeText("192.168.1.105", agentFile);
+        Utils.writeText(AGENT_IP_ADDRESS, agentFile);
```

Comment:
I think we should find a better mechanism to determine a IP address. I don't like this defaulting thing very much; although it is less bad than the hard coded IP address.